apis/Google.Cloud.Firestore/Google.Cloud.Firestore/Converters/EnumConverter.cs

```diff
@@ -26,94 +26,65 @@ namespace Google.Cloud.Firestore.Converters
     /// </summary>
     internal sealed class EnumConverter : ConverterBase
     {
-        private static readonly ConcurrentDictionary<BclType, PrimitiveConverter> s_primitiveConverters =
-            new ConcurrentDictionary<BclType, PrimitiveConverter>();
-
-        private readonly PrimitiveConverter _converter;
+        private TypeCode typeCode;
```

Comment:
Could you rename this to `_typeCode` and make it readonly please?

Reply:
Woops, good catch.